ChuckNorrisFacts/Presentation/List Facts/ListFactsViewModel.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ListFactsViewModel.swift
+//  ChuckNorrisFacts
+//
+//  Created by Munir Wanis on 21/05/18.
+//  Copyright © 2018 Wanis. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+
+class ListFactsViewModel {
+    let service = FactService()
```

Comment:
Se você colocar o FactService pra ser recebido no init desse view model, vc vai deixar ele melhor pra testar também!
Porque aí você vai conseguir fazer o mock do `FactService` e forçar ele a ter o comportamento que vc quiser testar no View Model

Reply:
O ViewModel também não era da etapa 🙃

Eu já fiz isso na branch da etapa 4 inclusive.

Reply:
Ahhhh! hahaha show então